Tella/Data/Networking/Repositories/UwaziServerRepositories.swift

```diff
@@ -139,6 +139,23 @@ class UwaziServerRepository: WebRepository {
             }
             .eraseToAnyPublisher()
         }
+    
+    func getRelationshipEntities(serverURL: String, cookie: String, templatesIds: [String?]) -> AnyPublisher<[UwaziRelationshipList], APIError> {
```

Comment:
do you think that` templatesIds: [String?]` are optionals unnecessarily?

Reply:
@ValbuenaG maybe `templatesId` could be an empty string but not a nil string ? so in that case  `templatesIds: [String]` could contains an empty string `""`

Reply:
done!